docs/topics/cli-args.adoc

```diff
@@ -24,6 +24,13 @@ To run the {ProductShortName} command without prompting, for example when execut
 |Argument |Description
 |--additionalClassPath |A space-delimited list of additional JAR files or directories to add to the class path so that they are available for decompilation or other analysis.
 |--addonDir |Add the specified directory as a custom add-on repository.
+|--analyzeKnownLibraries | Analyze known software artifacts embedded within your application. By default {ProductShortName} only analyzes application code.
+
```

Comment:
@RichardHoch  
All arguments in this table used for choosing existing options are called flags ("Flag to..."). Why isn't this one a flag? "Flag to analyse known..."
Also, a comma after "By default".